### PR TITLE
Update rbuilder version and refactor image dependencies 

### PR DESCRIPTION
**Pull request by MoeMahhouk**
This PR does the following:
- update the rbuilder to the version where I added the parsing of CL nodes from env variable/cloud init config.
- decouples the the meta-confidential-compute layer by appending the rbuilder to the image when adding the meta-evm layer.
This way, no need to manually change the dependencies in initramfs from within the meta-confidential-layer everytime. 
In other words, modularity and better maintainability when developing for different purposes as we are doing in the BOB searcher project

### CODE REVIEW

File: recipes-core/images/cvm-initramfs.bbappend [hunk 1]
```diff
@@ -0,0 +1,2 @@
+PACKAGE_INSTALL:append = " rbuilder"
+IMAGE_FEATURES:append = "debug-tweaks"
```
Comment (Ruteri) [inline recipes-core/images/cvm-initramfs.bbappend:2]: Is this needed?

Comment (MoeMahhouk) [reply recipes-core/images/cvm-initramfs.bbappend:2]: actually, I added this here but I was thinking to create a branch for the dev purposes.
This should not be there for the final production deployment. But this is needed for having ssh access for debugging purposes at the moment.
I can add a comment above it as TODO: remove when productionizing/finalizing the rbuilder + yocto image

Comment (Ruteri) [reply recipes-core/images/cvm-initramfs.bbappend:2]: Could we maybe have a separate "debug" bbappend that's included or something?

**Review by Ruteri — APPROVED** [review]

File: recipes-core/images/cvm-initramfs.bbappend (hunk 1 above)
Comment (MoeMahhouk) [reply recipes-core/images/cvm-initramfs.bbappend:2]: We could do something similar to the other PRs where we have the python script fetching if DEBUG was enabled and append this debug-tweaks to the image conditionally.
How does this sound?

Comment (MoeMahhouk) [reply recipes-core/images/cvm-initramfs.bbappend:2]: I experimented a bit with this and I think I can move this back to the meta-confidential layer and have there the debug value to be set from an env variable prior to building.
I managed to make that work

**Pull request merged**